### ISSUE TITLE
Mention WHICH label occurs multiple times

### ISSUE THREAD

**Issue by NorfairKing** [typst/typst#2454]
### Description

I see this error:
```
error: label occurs multiple times in the document
    ┌─ spec/main.typ:146:5
    │  
146 │     [ #link(
    │ ╭──────^
147 │ │       label("event_" + event.identifier),
148 │ │       [ #raw(event.acronym) (#text(event.name)) ],
149 │ │     ) ]
    │ ╰─────^
```
However: Label is used programmatically, and now I have no clue which label it is that occurs multiple times.

### Use Case

Debugging.

**Comment by PgBiel** [comment]
As a workaround, you can place a `locate` block right before your `#link` where you `query(label("event_" + event.identifier), loc)` and then check if the array returned by query has `.len() > 1`.

**Comment by laurmaedje** [comment]
Should be an easy fix. There've already been a few PRs for similar diagnostics that are uninformative in scripted contexts.

**Comment by tingerrr** [comment]
Resolved as of 55296a919.

**Comment by NorfairKing** [comment]
Woah, thanks!
